backend/src/services/perplexity_response_parser.py

```
@@ -28,8 +28,11 @@ def parse_restaurants(response: str) -> List[Dict]:
             if not line:
                 continue
             
-            # Check for numbered restaurant with inline format: "1. **Name** - Address"
-            inline_match = re.match(r'^(\d+)\.\s+\*\*(.+?)\*\*\s*[-–]\s*(.+)', line)
+            # Check for numbered restaurant with inline format: 
+            # "1. **Name** - Address"
+            inline_match = re.match(
+                r'^(\d+)\.\s+\*\*(.+?)\*\*\s*[-–]\s*(.+)', line
```

Ruff (0.13.1)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)